android-client-sdk/src/test/java/com/devcycle/sdk/android/api/DVCClientTests.kt

```diff
@@ -271,6 +271,7 @@ class DVCClientTests {
         client.close()
     }
 
+    @ExperimentalCoroutinesApi
```

Comment:
Are there any tests we can add for request retrying?

Reply:
I can see what I can do but it would require a lot of mocking setup and would serve as a integration test better